clients/mobile/src/setup/navigation/routes.ts

```diff
@@ -69,5 +71,11 @@ export const routes = [
     component: ResponseDonationRequest,
     options: { headerShown: true, headerTitle: 'Blood Request' },
     protected: true
+  },
+  {
+    name: SCREENS.DONAR_RESPONSE,
+    component: DonorResponse,
+    options: { headerShown: true, headerTitle: '' },
```

Comment:
Why is the headerTitle empty?

Reply:
For this screen, there is no header title that's why it's empty.

Reply:
got it